cmd/tempo/main.go

```diff
@@ -73,6 +74,10 @@ func main() {
 		}
 	}()
 
+	if *mutexProfileFraction > 0 {
```

Comment:
```suggestion
	if *mutexProfileFraction > 0  && *mutexProfileFraction <= 1 {
```
protect against bad fractions?

Reply:
`mutexProfileFraction` is an int and is the denominator of the fraction. i.e. `5` is acceptable input and means that 1/5 mutex interactions are profiled.

Reply:
Mmm, that's confusing, would be more intuitive if it had `-denominator` in the flag name